src/views/Documents.vue

```diff
@@ -3,7 +3,7 @@
 		class="bg-gray-200"
 		data-e2e-id="documentsPage"
 	>
-		<div class="container mx-auto mt-10">
```

Comment:
Table headers seems to be broken here.. we could decrease the width/padding/margin, couldn't we?

![image](https://user-images.githubusercontent.com/27869979/98367029-3cc79380-2035-11eb-94e2-58beb6e7a898.png)